base/array.jl

```diff
@@ -879,6 +879,18 @@ function lexcmp(a::Array{UInt8,1}, b::Array{UInt8,1})
     return c < 0 ? -1 : c > 0 ? +1 : cmp(length(a),length(b))
 end
 
+# use memcmp for == on bit integer types
+=={T<:BitInteger,N}(a::Array{T,N}, b::Array{T,N}) =
+    size(a) == size(b) &&
+    ccall(:memcmp, Int32, (Ptr{T}, Ptr{T}, UInt), a, b, sizeof(T) * length(a)) == 0
```

Comment:
It just hides the status. It's probably already started by now and should still post a comment, and if the code is functionally the same and your rebase was just a style change, shouldn't matter.